snapshotter/cmd/devmapper/main.go

```diff
@@ -30,10 +34,52 @@ const (
 )
 
 func main() {
-	var configPath string
+	var (
+		ctx        = context.Background()
+		config     = &devmapper.Config{}
+		configPath = ""
+		rootPath   = ""
+	)
+
 	flag.StringVar(&configPath, "config", "", "Path to devmapper configuration file")
+	flag.StringVar(&rootPath, "path", "", "Path to snapshotter data")
+
+	// These flags are needed for compatibility with container-storage-setup tool as there is no way to
+	// ignore unknown flags in `flag` package. Storage options are parsed using visitKeyValueOpts func.
```

Comment:
You can use a `flag.FlagSet` and change the `flag.ErrorHandling` to use `flag.ContinueOnError`.

Reply:
These lines are needed to prevent errors like `flag provided but not defined: -storage-opt`.
With `FlagSet` I still need to register them, so that wouldn't help.

Reply:
Discussed offline; using a FlagSet requires more refactoring and we can come back to that later.